src/Form/Type/FormatterType.php

```diff
@@ -44,39 +41,18 @@ final class FormatterType extends AbstractType
     protected $translator;
 
     /**
-     * @var ConfigManagerInterface
+     * @var CKEditorConfigurationInterface
      */
-    protected $configManager;
-
-    /**
-     * @var PluginManagerInterface
-     */
-    protected $pluginManager;
-
-    /**
-     * @var TemplateManagerInterface
-     */
-    private $templateManager;
-
-    /**
-     * @var ToolbarManagerInterface
-     */
-    private $toolbarManager;
-
+    protected $ckEditorConfiguration;
```

Comment:
I guess this doesn't need to be protected since the class is declared as final.
```suggestion
    private $ckEditorConfiguration;
```

Reply:
done